docs/source/model_doc/gpt2.rst

```diff
@@ -114,3 +114,9 @@ TFGPT2DoubleHeadsModel
 
 .. autoclass:: transformers.TFGPT2DoubleHeadsModel
     :members: call
+
+TFGPT2ForSequenceClassification
```

Comment:
Could you also add the `TFSequenceClassifierOutputWithPast` here in the model-specific outputs?

Reply:
Ok I'll do it.

Reply:
Done.